server/sonar-server/src/main/java/org/sonar/server/telemetry/TelemetryDataJsonWriter.java

```diff
@@ -34,16 +34,42 @@ public static void writeTelemetryData(JsonWriter json, TelemetryData statistics)
     json.beginObject();
     json.prop("id", statistics.getServerId());
     json.prop("version", statistics.getVersion());
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "version" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1192)